README.md

```diff
@@ -66,6 +66,7 @@ If you are not on mac, please do this extra step and install these libraries as
 npm install --save-dev @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers
 ```
 
+- Before creating a new contract, let's delete the contract `Lock.sol` inside `contracts` directory.
```

Comment:
Not necessary

Reply:
If the file is not deleted, the compilation will fail.

Reply:
interesting, i'll try it out and get back to  you

Reply:
In another repo, NFT-collection, the same error occurs.

![image](https://user-images.githubusercontent.com/6451384/178958369-f6f278b9-a68d-408f-b6a7-1ea1e2982662.png)